src/org/openlcb/swing/NodeSelector.java

```diff
@@ -144,7 +166,9 @@ public String toString() {
                 justification = "Purposefully attempting lookup using NodeID argument in model " +
                         "vector.")
         public boolean equals(Object o) {
-            if (o == null) return false;
+            if (o == null) {
+                return false;
+            }
```

Comment:
This addition is causing a spot bugs error in the build:

[INFO] org.openlcb.swing.NodeSelector$ModelEntry defines equals and uses Object.hashCode() [org.openlcb.swing.NodeSelector$ModelEntry] At NodeSelector.java:[lines 169-178] HE_EQUALS_USE_HASHCODE
335[INFO] 

The next line Includes an instanceof operator.  Instanceof returns false for null, so the null case is already handled.

Reply:
Removed null check in equals() as requested. Also added hashCode() to potentially fix spot bugs warning.
Could you please give me a hint on how to execute spot bugs to prevent such warnings on future changes?

Reply:
Spogbugs is run on travis using the following Maven command:

`mvn clean verify -U -P spotbugs --batch-mode -Dmaven.javadoc.skip=true`